compose/compose-file/index.md

```diff
@@ -691,6 +699,23 @@ If you have further questions, please refer to the discussion on the GitHub
 issue [docker/compose/4513](https://github.com/docker/compose/issues/4513){: target="_blank" class="_"}.
 {: .important}
 
+##### Generic Resources
+
+Generic resources are a way to select the kind of nodes your task can land on.
+
+In a swarm cluster, nodes can advertise Generic resources as Discrete values or
+as Named values such as SSD=3 or GPU=UID1, GPU=UID2, ...
```

Comment:
No ellipsis here (just end your list with `or GPU=UID1`), and maybe mark up the key-value pairs with back-ticks to make them show as mono text.